src/main/java/xbot/common/command/BaseRobot.java

```diff
@@ -253,4 +267,26 @@ else if(timeSinceLastLog >= frequencyReportInterval.get()) {
         
         outsidePeriodicMonitor.start();
     }
+
+    
+    @Override
```

Comment:
Azure is complaining about these @Override but they are fine locally

Reply:
When I do a clean build of SCL locally, I hit the same error. Something is probably referencing an older version of WPILib.

Reply:
Yeah, everything in SCL is pointing to 2020.1.2 versions; those probably need to be incremented.